### PR TITLE
ADD GO CODE!!!!!!!! 🥰😍🔥✨

### PR DESCRIPTION
**Pull request by pjebs**
I contributed the PHP code in readme. Now I'm a go dev.

### CODE REVIEW

**Comment by pjebs** [comment]
@martenson can this possibly get merged?

File: README.md [hunk 1]
```diff
@@ -43,6 +43,30 @@ function isDisposableEmail($email, $blocklist_path = null) {
...
+}
+
+func isDisposableEmail(email string) (disposable bool) {
+	segs := strings.Split(email, "@")
+	_, disposable = disposableList[strings.ToLower(segs[len(segs)-1])]
+	return
+}
+```
+
+Alternatively, [anti-disposable-email](https://github.com/rocketlaunchr/anti-disposable-email) which auto-updates based on this repo.
```
Comment (martenson) [inline README.md:67]: I would rather not endorse a project with a custom license (https://github.com/rocketlaunchr/anti-disposable-email/blob/master/LICENSE)

Comment (pjebs) [reply README.md:67]: It's practically the MIT license with an extra clause.

Comment (martenson) [reply README.md:67]: It is practically NOT the MIT license.

Comment (pjebs) [reply README.md:67]: Okay I'll change it to MIT

Comment (pjebs) [reply README.md:67]: Done

Comment (pjebs) [reply README.md:67]: @martenson will this get accepted?

File: README.md [hunk 2]
```diff
@@ -43,6 +43,30 @@ function isDisposableEmail($email, $blocklist_path = null) {
     return in_array($domain, $disposable_domains);
 }
 ```
+
+**Go** contributed by [@pjebs](https://github.com/pjebs)
+
+```go
+import ("bufio"; "fmt"; "os"; "strings";)
```
Comment (martenson) [inline README.md:50]: how is the `fmt` import used?

Comment (pjebs) [reply README.md:50]: You're right. Let me remove it.

Comment (pjebs) [reply README.md]: @martenson fixed.

**Comment by pjebs** [comment]
@martenson any more changes or is it all good?

**Comment by martenson** [comment]
@pjebs thanks for the contribution!

**Pull request merged**